suite-native/atoms/src/Input/Input.tsx

```diff
@@ -67,25 +71,32 @@ type InputLabelStyleProps = {
 };
 
 type InputStyleProps = {
+    isLabelDisplayed: boolean;
     isLeftIconDisplayed: boolean;
     isRightIconDisplayed: boolean;
     isDisabled: boolean;
 };
 
 const inputWrapperStyle = prepareNativeStyle<InputWrapperStyleProps>(
-    (utils, { hasError, hasWarning, isFocused, elevation }) => ({
-        backgroundColor: utils.colors.backgroundNeutralSubtleOnElevation0,
-        borderColor: utils.colors.backgroundNeutralSubtleOnElevation1,
+    (utils, { hasError, hasWarning, isDisabled, isFocused, elevation }) => ({
+        backgroundColor: isDisabled
+            ? utils.colors.backgroundNeutralSubtleOnElevation1
+            : utils.colors.backgroundNeutralSubtleOnElevation0,
+        borderColor: isDisabled
+            ? utils.colors.backgroundNeutralSubtleOnElevation1
+            : utils.colors.borderInputDefault,
```

Comment:
It doesn't respect elevation semantics btw

Reply:
This was discussed with @shenkys, the border colour needs to match the background colour when disabled. There will be some UI revision with @PeKne AFAIK. If you believe it should be done differently, let's discuss in person. 🙏